src/packages/recompose/__tests__/pick-test.js

```diff
@@ -0,0 +1,29 @@
+import test from 'ava'
+import pick from '../utils/pick'
+
+test('pick', t => {
+  t.deepEqual(
+    pick({
+      one: '1',
+      dont: 'include',
+      two: {
+        deep: {
+          value: 'true'
+        }
+      },
+      root: {},
+      three: '3'
+    }, [
+      'one',
+      ['two', 'deep', 'value'],
+      ['root', 'missed'],
+      'three'
+    ]),
+    {
+      one: '1',
+      'two\ndeep\nvalue': 'true',
```

Comment:
Seperator could be changed if we don't like the new line but property names probably won't include new lines (that is why I choose that character).